src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json

```diff
@@ -0,0 +1,127 @@
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "contact_point": {
+        "name": "Subnational Development Team",
+        "email": "mailto:Subnational.Development@ons.gov.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/business-trade-and-international-development"
+      ],
+      "frequency": "annual",
+      "keywords": [
+        "public-funded",
+        "research-and-development",
+        "expenditure" ,
+        "subnational",
+        "region"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-10-27T10:41:11Z",
+      "modified": "2023-11-02T14:41:11Z",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1Y"
+      ],
+      "temporal_coverage": {
+        "start": "2022-01-01T09:00:00Z",
+        "end":  "2022-12-31T09:00:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/public-funded-gross-expenditure/editions/2023-10/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/public-funded-gross-expenditure/editions/2023-10/versions/1",
+          "issued": "2023-10-27T10:41:11Z"
+        }
+      ],
+      "next_release": "2024-10-01T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Year",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name associated with the geography code."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The geography code associated with the observation."
+          },
+          {
+            "name": "geography",
+            "datatype": "string",
+            "titles": "Geography Type",
+            "description": "The categorisation of the geography code."
+          },
+          {
+            "name": "observation",
+            "datatype": "decimal",
+            "titles": "Gross disposable household income per head",
```

Comment:
The observation refers to gross disposable income, but the dataset is for gross for expenditure on R&D

Reply:
Noted

Reply:
resolved